### PR TITLE
feat(taiko-client): skip L2 blob tx when proposing a L2 block

### PR DESCRIPTION
**Pull request by YoGhurt111**

### CODE REVIEW

File: packages/taiko-client/driver/txlist_decompressor/txlist_decompressor.go [hunk 1]
```diff
@@ -72,6 +73,16 @@ func (v *TxListDecompressor) TryDecompress(
 		return []byte{}
 	}
 
+	// Re-encode txs when txs contains invalid tx
+	for _, tx := range txs {
+		// Skip invalid L2 blob tx
+		if tx.Type() == types.BlobTxType {
```
Comment (smtmfft) [inline packages/taiko-client/driver/txlist_decompressor/txlist_decompressor.go:79]: if tx.Type() == types.BlobTxType  && fork <= SHANGHAI ??

Comment (YoGhurt111) [reply packages/taiko-client/driver/txlist_decompressor/txlist_decompressor.go:79]: We won't support l2 blob tx after cancun either.

Comment (mask-pp) [reply packages/taiko-client/driver/txlist_decompressor/txlist_decompressor.go:79]: filter blob tx belongs to `taiko-geth` function, it seems inappropriate to decide how geth should do.

Comment (davidtaikocha) [reply packages/taiko-client/driver/txlist_decompressor/txlist_decompressor.go:79]: Actually don't think this is necessary? taiko-geth will handle it?

Comment (YoGhurt111) [reply packages/taiko-client/driver/txlist_decompressor/txlist_decompressor.go:79]: If taiko-geth will do this case, taiko-geth has to do it now.
It is a breaking bug and needs to be quickly fixed.

**Comment by davidtaikocha** [comment]
Closed due to we want to apply it in taiko-geth

**Pull request closed without merging**